tests/kas_messaging.spec.ts

```diff
@@ -104,3 +106,31 @@ test('Browse messages', async ({ page }) => {
   await expect(messageDetail.locator('dt:has-text("Offset")')).toHaveCount(1);
   await expect(messageDetail.locator('dd:has-text("key-")')).toHaveCount(1);
 });
+
+// test_6acl.py test_kafka_create_consumer_group_and_check_dashboard
+test('create consumer group and check dashboard', async ({ page }) => {
+  const instanceLinkSelector = page.getByText(testInstanceName);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+
+  await navigateToKafkaList(page);
+  await waitForKafkaReady(page, testInstanceName);
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Connection').click();
+
+  const bootstrapUrl = await getBootstrapUrl(page, testInstanceName);
+  console.log('bootstrapUrl: ' + bootstrapUrl);
```

Comment:
note for the future:
should we start using a proper logging framework?

Reply:
Yes!

Reply:
sure @andreaTP

Reply:
rebased